ReactiveCocoa/Swift/SignalProducer.swift

```diff
@@ -804,6 +804,59 @@ public func timer(interval: NSTimeInterval, onScheduler scheduler: DateScheduler
 	}
 }
 
+/// Creates a repeating timer of the variable interval starting with `nextInterval(0)`,
+/// with a reasonable default leeway, sending updates on the given scheduler.
+///
+/// This timer will complete when `nextInterval` returns nil.
```

Comment:
I don't think it's worth introducing yet another way of completing a producer :-1: